### PR TITLE
feat: System Bars & Display Density

### LINKED ISSUE #362: Add Get Display Density endpoint
## Description
Implement Get Display Density endpoint to permit retrieving display density(dpi) of the Android device

## Endpoint Documentation
http://appium.io/docs/en/commands/device/system/display-density/

### PR DESCRIPTION
**Pull request by akinsolb**
closes #362 & #363

## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [x] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [x] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details
Add API for System bars and Get Display Density

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/Appium.Net/Appium/Android/AndroidDriver.cs [hunk 1]
```diff
@@ -211,6 +211,19 @@ public void SetGsmSignalStrength(GsmSignalStrength gsmSignalStrength)
...
         #endregion
 
+        #region Device System
+
+        /// <summary>
+        /// Open the notifications 
+        /// </summary>
+        public void OpenNotifications() => AndroidCommandExecutionHelper.OpenNotifications(this);
+
+        public IDictionary<string, object> GetSystemBars() => AndroidCommandExecutionHelper.GetSystemBars(this);
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Android/AndroidDriver.cs:221]: it would be useful to have documentation on these methods

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:221]: good spot. done

File: src/Appium.Net/Appium/Android/AndroidDriver.cs [hunk 2]
```diff
@@ -211,6 +211,27 @@ public void SetGsmSignalStrength(GsmSignalStrength gsmSignalStrength)
...
+
+        /// <summary>
+        /// Open the notifications 
+        /// </summary>
+        public void OpenNotifications() => AndroidCommandExecutionHelper.OpenNotifications(this);
+
+        /// <summary>
+        /// Retrieve visibility and bounds information of the status and navigation bars
+        /// </summary>
+        /// <returns>A dictionary whose keys are <code>statusBar</code><code>navigationBar</code></returns>
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Android/AndroidDriver.cs:224]: could we also mention the expected types of dict values?

**Review by mykola-mokhnach — APPROVED** [review]

File: src/Appium.Net/Appium/Android/AndroidDriver.cs (hunk 2 above)
Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs]: Sure. change has been made.

**Pull request merged**